health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L1Main.js

```
@@ -0,0 +1,689 @@
+      {response?.responseData?.data?.[0]?.insight?.value ? <Insight /> : null}
+    </div>
+  );
+};
+
+const HorBarChart = ({ data, setselectState = "" }) => {
+  const barColors = ["#298CFF", "#54D140"];
+  const { t } = useTranslation();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  const { id, chartType } = data;
+  let filters = {};
+  const selectedProjectTypeId = projectTypeId ? projectTypeId : Digit.SessionStorage.get("selectedProjectTypeId");
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)